website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Biome v1.7
+description: Migrate from Prettier and ESLint with one command!
+summary: |
+  This new version provides an easy path to migrate from ESLint and Prettier.
+  It also introduces machine-readable reports for the formatter and the linter, new linter rules, and many fixes.
+authors:
+  - conaclos
+  - team
+pubDate: 2024-04-15
+coverImage:
+  lightSrc: "@/assets/blog/roadmap-2024/banner-light.png"
+  darkSrc: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we’re excited to announce the release of _Biome v1.7_!
```

Comment:
I agree with Arend here. The italic is a bit excessive, it's not needed. Can we remove it from all the names? Biome, Eslint, etc.